src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,51 @@
 import React from 'react';
+import { Post } from '../../types/types';
+import { CommentList } from '../CommentList';
+import usersFromServer from '../../api/users';
+import commentsFromServer from '../../api/comments';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+function filterCommets(comments: Comment[], postOfId: number) {
```

Comment:
It's recommended to fix the typo in the function name `filterCommets` to `filterComments` for better readability.